### PR TITLE
test: Add step to close onboarding modals

### PR DESCRIPTION
**Pull request by SamuelSalas**
## **Description**

With the latest app builds, multiple onboarding modals should be displayed after the importing the wallet. The objective is to implement the same steps already written on Detox into Appium, for the automation test to handle them and to continue with the next steps.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

Upgrading test run: https://app.bitrise.io/build/a90fcdbf-ddda-4036-8873-1aa4653a58ee
Release e2e test run: https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/74b6c544-d248-4ed8-a762-1a822110f62b

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: wdio/config/android.config.browserstack.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ config.waitforTimeout = 10000;
 config.connectionRetryTimeout = 90000;
 config.connectionRetryCount = 3;
 config.cucumberOpts.tagExpression =
-  process.env.CUCUMBER_TAG_EXPRESSION || '@performance and @androidApp'; // pass tag to run tests specific to android
+  process.env.CUCUMBER_TAG_EXPRESSION || '@upgrade and @androidApp'; // pass tag to run tests specific to android
```
Comment (cortisiko) [inline wdio/config/android.config.browserstack.js:33]: this is basically stating that the app upgrade test should run by default. My question is, what happens to the warm and cold start tests?

**Review by cortisiko — CHANGES_REQUESTED** [review]
Everything looks good i just left one small comment

File: wdio/config/android.config.browserstack.js (hunk 1 above)
Comment (SamuelSalas) [reply wdio/config/android.config.browserstack.js:33]: I made this change just for local purposes. I'll revert it in the next commit.

**Review by cortisiko — APPROVED** [review]
✅ ✅ ✅

**Pull request merged**